src/RGUnityBots/Assets/Scripts/FPSCap.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+public class FPSCap : MonoBehaviour
+{
+    public int targetFrameRate = 30;
+
+    private void Start()
+    {
+        QualitySettings.vSyncCount = 0;

```

Comment:
This is here to limit the amount of laptop fan noise while testing this input project as we really don't need it running at max fps/cpu